client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/faqs.tsx

```diff
@@ -0,0 +1,148 @@
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import FoldableFAQ from 'calypso/components/foldable-faq';
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
+import type { FC } from 'react';
+
+import './styles.scss';
+
+const DomainTransferFAQ: FC = () => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const siteId = useSelector( getSelectedSiteId );
+
+	const onFaqToggle = useCallback(
+		( faqArgs: { id: string; buttonId: string; isExpanded: boolean; height: number } ) => {
+			const { id, buttonId, isExpanded } = faqArgs;
+			const tracksArgs = {
+				site_id: siteId,
+				faq_id: id,
+			};
+
+			const removeHash = () => {
+				history.replaceState( '', document.title, location.pathname + location.search );
+			};
+
+			// Add expanded FAQ buttonId to the URL hash
+			const addHash = () => {
+				history.replaceState(
+					'',
+					document.title,
+					location.pathname + location.search + `#${ buttonId }`
+				);
+			};
+
+			if ( isExpanded ) {
+				addHash();
+				// FAQ opened
+				dispatch( recordTracksEvent( 'calypso_domain_transfer_faq_open', tracksArgs ) );
+			} else {
+				removeHash();
+				// FAQ closed
+				dispatch( recordTracksEvent( 'calypso_domain_transfer_faq_closed', tracksArgs ) );
+			}
+		},
+		[ siteId, dispatch ]
+	);
+
+	return (
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
+		<>
+			<section>
+				<h2>{ translate( 'Frequently Asked Questions' ) }</h2>
+				<ul>
+					<li>
+						<FoldableFAQ
+							id="service-down"
+							question={ translate(
+								'Will my website or email service be down during the transfer?'
+							) }
+							onToggle={ onFaqToggle }
+							className="domain-transfer-faq__section"
+						>
+							{ translate(
+								'If your website and email settings were correctly configured before the transfer, there should be no downtime.'
+							) }
+						</FoldableFAQ>
+					</li>
+					<li>
+						<FoldableFAQ
+							id="time-left"
+							question={ translate(
+								'Do I lose the time left with my current registrar when I transfer?'
+							) }
+							onToggle={ onFaqToggle }
+							className="domain-transfer-faq__section"
+						>
+							{ translate(
+								'No, when you transfer your domain, the remaining time you had left with your old registrar ' +
+									'is transferred over. In addition, we give you another free years registration when you ' +
+									'transfer from Google Domains.'
```

Comment:
We should make this more agnostic since the flow is not only for Google Domains.

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/79503/commits/e9d3cc8d798ef1e3abadf7eb3e382de9ff29883e